src/passes/RoundTrip.cpp

```diff
@@ -39,28 +31,18 @@ namespace wasm {
 
 struct RoundTrip : public Pass {
   void run(PassRunner* runner, Module* module) override {
-    std::string templateName = "byn_round_trip_XXXXXX";
-    std::vector<char> buffer(templateName.begin(), templateName.end());
-    buffer.push_back(0);
-#ifndef _WIN32
-    auto fd = mkstemp(buffer.data());
-    WASM_UNUSED(fd);
-    std::string tempName(buffer.begin(), buffer.end());
-#else
-    std::string tempName = _mktemp(buffer.data());
-#endif
-    // Write
-    ModuleWriter writer;
-    writer.setBinary(true);
-    writer.setDebugInfo(runner->options.debugInfo);
```

Comment:
do we not need to set debug info on the writer any more?

Reply:
No, `WasmBinaryWriter` emits the debuginfo by default, unlike `ModuleWriter`. The `dwarfdump_roundtrip_dwardump.wast` test will fail if this changes.